_posts/2022-11-07-road-to-quarkus-3.adoc

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: 'Starting Quarkus 3'
+date: 2022-11-07
```

Comment:
Once scheduled, this date needs to be changed (and the date in the filename too).